### DISCUSSION TITLE
[Feature Request]: CookieCloud不如以前用油猴拿密钥传站点信息

### DISCUSSION THREAD

**Discussion by Adsryen** [jxxghp/MoviePilot#519] [建议]
### 当前程序版本

1.1.5

### 功能改进

很多站点ptpp都不一定能更新到cookie进去，就别说CookieCloud了，以前用nt的时候，我80+的站点，将近40+都是手动进的网站用油猴存进来的，前几天用CookieCloud更新了浏览器的cookie，一样的很多站点都存不进去，希望以后有更好的解决方式吧

**Comment by jxxghp** [comment]
MP提供了新增站点的API，可以通过油候脚本添加。
